### PR TITLE
[KARAF-6953] Introduce autoRefresh property in etc/org.apache.karaf.features.cfg

### PR DESCRIPTION
**Pull request by jbonofre**

### CODE REVIEW

File: features/core/src/main/java/org/apache/karaf/features/FeaturesService.java [hunk 1]
```diff
@@ -42,6 +42,7 @@
     long DEFAULT_SCHEDULE_DELAY = 250;
     int DEFAULT_SCHEDULE_MAX_RUN = 9;
     long DEFAULT_REPOSITORY_EXPIRATION = 60000; // 1 minute
+    boolean DEFAULT_AUTO_REFRESH = false;
```
Comment (daniellas) [inline features/core/src/main/java/org/apache/karaf/features/FeaturesService.java:45]: Does it change current default setting?

**Comment by k-brooks** [comment]
Thanks!

**Comment by jbonofre** [comment]
I handle the `deploy` phase, I will do the same for the `undeploy` phase.

**Comment by ahgittin** [comment]
Nice one!  Looking forward to this.

**Pull request merged**